tests/components/ResultsCount.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Source, State } from '@yext/answers-headless-react';
+import { render, screen } from '@testing-library/react';
+import { ResultsCount } from '../../src/components/ResultsCount';
+import { mockAnswersState } from '../__utils__/mocks';
+
+
+const mockedStateUniversal: Partial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'universal'
+  },
+  universal: {
+    verticals: [{
+      appliedQueryFilters: [],
+      queryDurationMillis: 2,
+      results: [],
+      resultsCount: 2,
+      source: Source.Google,
+      verticalKey: 'string'
+    },
+
```

Comment:
remove new lines! (eslint did show errors here?)

Reply:
alex also didn't see eslint complain here, feel free to make a note here!

Reply:
https://yexttest.atlassian.net/browse/SLAP-2135

Reply:
I'm not sure if there's a rule for this though

Reply:
updated the item to include this example! I'm also not sure if there's one but it's worth doing a quick scan through their rules about lists and objects styling